internal/handlers/season.go

```diff
@@ -4,9 +4,42 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	"github.com/vultisig/airdrop-registry/internal/utils"
 )
 
-func (a *Api) getCurrentSeasonInfo(c *gin.Context) {
-	currentSeason := a.cfg.Season
-	c.JSON(http.StatusOK, currentSeason)
+func (a *Api) getAllSeasonInfo(c *gin.Context) {
+	allSeasons := a.cfg.Seasons
+	c.JSON(http.StatusOK, allSeasons)
+}
+
+type SeasonPoints struct {
+	Points float64 `json:"points"`
+}
+
+func (a *Api) getTotalPointsBySeasonHandler(c *gin.Context) {
+	startId := uint(0)
+	totalPoints := 0.0
+	for {
+		allVaults, err := a.s.GetVaultsWithPage(startId, 1_000)
```

Comment:
You should think about improve this , because this calculate the total point per user's request , which is quite costly , instead we should calculate it daily , and then save it to db , and then just return that.